docs/develop/howtos/override_components.md

```diff
@@ -0,0 +1,121 @@
+# How to override a react component
+
+This documentation is targeted to developers that want to customize a specific react component in an instance.
+
+## Overridable
+
+It is possible to override certain react components in Invenio RDM, by using the library [`react-overridable`](https://github.com/indico/react-overridable). This feature is available to instance developers to customize the look and feel of the react application (e.g. deposit form).
+
+Despite the library allowing multiple ways of overriding a component, for this guide we will implement a new component to replace the default one.
+
+## Steps
+
+- Make sure that the component you want to override is implemented in the same level as an `override.js` file. For example, for the deposit form, `invenio-app-rdm` has the following structure:
+
+```terminal
+├── invenio_app_rdm
+│   ├── assets
+│   │   ├── semantic-ui
+│   │   |   ├── js
+│   │   |   |   ├── invenio_app_rdm
+│   │   |   |   |   ├── deposit
+│   │   |   |   |   |   ├── override.js
+│   │   |   |   |   |   ├── RDMDepositForm.js
+```
+
+We will override the file `override.js`.
+
+- Find a react component to override. For this example, we are going to override the deposit form '`Metadata-only record`' checkbox.
+
+!['`Metadata-only record`' checkbox](./img/metadata_only_checkbox.png)
+
+!!! info "You must know the component id beforehand"
+    In order to override a component, the exact overridable id must be retrieved beforehand.
+    For now, you must retrieve the component id directly from the module's code.
+
+The react component's overridable id for the '`Metadata-only record`' checkbox is `ReactInvenioDeposit.MetadataAccess.layout`.
```

Comment:
Info block is not rendering properly in the displayed documentation.
![Screenshot from 2023-01-16 11-11-05](https://user-images.githubusercontent.com/25476209/212653155-edbf7643-9800-4a3b-a1de-3bad63d41b26.png)